moveit_core/planning_request_adapter/src/planning_request_adapter.cpp

```diff
@@ -148,10 +149,12 @@ bool PlanningRequestAdapterChain::adaptAndPlan(const planning_interface::Planner
 
     // if there are adapters, construct a function pointer for each, in order,
     // so that in the end we have a nested sequence of function pointers that call the adapters in the correct order.
-    PlanningRequestAdapter::PlannerFn fn = boost::bind(&callAdapter1, adapters_.back().get(), planner, _1, _2, _3,
-                                                       boost::ref(added_path_index_each.back()));
+    PlanningRequestAdapter::PlannerFn fn =
+        std::bind(&callAdapter1, adapters_.back().get(), planner, std::placeholders::_1, std::placeholders::_2,
+                  std::placeholders::_3, boost::ref(added_path_index_each.back()));
```

Comment:
ok, this was missing in the original PR as well, no need to change it here

Reply:
sounds like a bug in @jspricke's sed scripts that converted the binds? I guess we should fix this instead of merging an inconsistent backport...
Also I still vote for replacing all binds with proper lambda closures, but someone would need to do a tiny bit more than a sed script to get there :)